### ISSUE TITLE
Narrator does not announce label associated with Spinner

### ISSUE THREAD

**Issue by yueou1995** [microsoft/fluentui#15601]
### Environment Information

Windows Version - Win10
Chromium Edge - Version 85.0.564.51 (Official build) (64-bit)

### Describe the issue:

Narrator does not announce label associated with Spinner

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/yueou/pen/qBNNzpJ

#### Actual behavior:

Narrator is not reading the spinner label

#### Expected behavior:

Narrator should read the spinner label

**Comment by paulgildea** [comment]
@yueou1995 Thanks for filing this issue.

@smhigley FYI
